samples/openthread/cli/snippets/low_power/boards/nrf54lm20dk_nrf54lm20a_cpuapp.overlay

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&adc {
```

Comment:
Have you checked that all of them are enabled by default, and if so, do we need to disable them here? Or do you want to ensure that they will always be disabled, regardless of the global configurations?

Reply:
In Matter we do not disable all of these things. But I don't know if it's good or bad, maybe that's why our power consumption is higher than expected. Would be nice to investigate it.